kinetic/distribution.yaml

```diff
@@ -6216,6 +6216,26 @@ repositories:
       url: https://github.com/ros-drivers/kvh_drivers.git
       version: master
     status: maintained
+  kvh_geo_fog_3d:
+    doc:
+      type: git
+      url: https://github.com/MITRE/kvh_geo_fog_3d.git
+      version: master
+    release:
```

Comment:
I get a 404 error when I try to look at the release repo https://github.com/MITRE/kvh_geo_fog_3d-release.git (is it a private repo?). If all that's desired is to generate documentation (and not generate debian packages) then the `release` section should be deleted.

Reply:
@zlacelle This is why it can't pull the package.xml

If you have not run bloom there should be no release stanza and you'll just have the `source` and `doc` stanzas for building from source.